src/main/MainTabsScreen.js

```diff
@@ -89,31 +89,48 @@ function MainTabsScreen(props: Props) {
           component={HomeScreen}
           options={{
             tabBarLabel: 'Home',
-            tabBarIcon: ({ color }) => <IconInbox size={24} color={color} />,
+            tabBarIcon: properties => (
+              <View style={styles.center}>
+                <IconInbox size={24} color={properties.color} />
+                {properties.focused && <Text style={styles.bottomNav}>Home</Text>}
+              </View>
+            ),
```

Comment:
Note there's already a label here: `tabBarLabel: 'Home'`.

Currently we don't usually show that label when on a phone, but we do on a tablet -- I see them on my iPad mini 4. I think if you create an emulated device that's an Android tablet, you'll see the same thing.

We should probably just control when those existing labels are shown. Certainly we should avoid showing double labels. :slightly_smiling_face:

Reply:
Sure, Greg! I didn't check in a tablet-sized-emulator so I thought it was something placed for accessibility, will use media queries and sort this!

Reply:
Done! Had to set showLabel to `false` in place of `Platform.isPad` BTW, I didn't see the labels on my (android) tablet emulator... does it only recognize Apple devices?